src/database/schema/ChannelListMemberSchema.ts

```diff
@@ -68,10 +68,15 @@ class ChannelListMemberSchema implements BaseDbSchema {
   };
 
   saveIfNotExist = async (db: SQLiteDatabase): Promise<void> => {
-    const ifExists = await this.checkIfExist(db);
-    if (ifExists) return;
+    try {
+      const ifExists = await this.checkIfExist(db);
+      if (ifExists) return;
 
-    this.save(db);
+      this.save(db);
+    } catch (e) {
+      console.log('saveIfNotExist error', e);
+      throw e;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `saveIfNotExist` function has been improved by adding a try-catch block. However, logging the error to the console might not be the best approach for production code. It would be better to use a dedicated logging service that can provide more context and handle different log levels. Also, rethrowing the error after logging it will cause the error to bubble up, which is good practice as it allows higher-level functions to decide how to handle the error.

```diff
-       console.log('saveIfNotExist error', e);
+       logger.error('saveIfNotExist error', e);
        throw e;
```
In this snippet, `logger` is an instance of a hypothetical logging service. You should replace it with whatever logging service you are using in your project.


<!-- This is an auto-generated comment by OSS CodeRabbit -->